src/public/modules/core/components/avatar-dropdown.client.component.js

```diff
@@ -83,7 +86,19 @@ function avatarDropdownController(
     },
   )
 
-  vm.signOut = () => Auth.signOut()
+  vm.logout = () => {
+    return $q
+      .when(AuthService.logout())
+      .then(() => {
+        // Clear contact banner on logout
+        $window.localStorage.removeItem('contactBannerDismissed')
+        // Redirect to landing page
+        $state.go('landing')
```

Comment:
maybe beyond the scope of this PR but we should really have these as constants somewhere